packages/slate-react/src/components/editable.tsx

```diff
@@ -114,6 +116,7 @@ export const Editable = (props: EditableProps) => {
     readOnly = false,
     renderElement,
     renderLeaf,
+    renderPlaceholder,
```

Comment:
I wanted to just make it a component. The reason I didn't was because I would have to use forwardRef to keep the [resizing logic](https://github.com/ianstormtaylor/slate/pull/4190/files#diff-18ef6af284404b1670857e562c183f2da6774d81bb65ed39d82fe124f769787bR46-R60) intact.